### PR TITLE
include vendor branching strategy for developers

### PR DESCRIPTION
**Pull request by axd1967**
### Description
See https://github.com/Stellarium/stellarium/discussions/1856

### CODE REVIEW

**Comment by alex-w** [comment]
In mid-summer I said that I'll disregard all future Alex's activity (reports, proposals, comments and suggestions) due his abusive behavior, but probably I should make an one exception right now and leave my comment here.

This proposal is strongly inappropriate for me as a maintainer and as developer, because this strategy introduces many problems and zero solutions, and this PR with "strategy docs" should be declined. A complete disregard of the opinion of the core developers and real maintainers of the Stellarium package isn't good behavior and experience.

**Comment by axd1967** [comment]
for anyone interested in "abusive behaviour" please see https://github.com/Stellarium/stellarium/issues/1804#issuecomment-885693896.

this proposal solves recurring issues that are explained in detail in https://github.com/Stellarium/stellarium/discussions/1856.

**Comment by axd1967** [comment]
"don't fix what ain't broken"... but there is no rule that prohibits improvements. in fact, that's part of Agile behaviour: continually reviewing and improving ways of work. and in this case, there are already signs that Stellarium will benefit from such a procedure. also, there is no obligation to use it; but there is a clear need to be aware of it.

File: VENDOR_BRANCHING.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Examples:
 - geonames data (https://www.geonames.org/recent-changes.html), https://github.com/Stellarium/stellarium-data/tags
 - quasar data (https://github.com/Stellarium/stellarium/blob/master/plugins/Quasars/util/quasars.tsv)
 - Almagest data (minor fixes, of course - this is essentially frozen data)
+- HTC algorithms (Helene, Telesto, and Calypso (Lagrangian satellites of Dione) - taken from ftp://ftp.imcce.fr/pub/ephem/satel/htc20/htc20.f ? )
```
Comment (gzotti) [inline VENDOR_BRANCHING.md:10]: Only in the unlikely case that we directly add their unmodified Fortran code to the project...

Comment (axd1967) [reply VENDOR_BRANCHING.md:10]: that's called Murphy's Law :-)

(and also, when one never tries, one will never become proficient in the pattern, so better try now than never)

**Comment by axd1967** [comment]
I'm clearly not really respected here, despite years of professional experience.

(@treaves @xalioth @Atque @Jocelyn1109 @guillaumechereau @A-j-K @abalkin )

**Comment by axd1967** [comment]
(related to [vendor_branching](https://github.com/Stellarium/stellarium/issues?q=vendor_branching))

**Pull request closed without merging**